tracer/build/_build/Build.GitHub.cs

```diff
@@ -1120,48 +1088,80 @@ async Task HideCommentsInPullRequest(int prNumber, string prefix)
                        .PullRequest(prNumber)
                        .Comments()
                        .AllPages()
-                       .Select(issue => new { issue.Id, issue.Body, issue.IsMinimized, });
-
-            var issueComments =  (await connection.Run(query)).ToList();
+                       .Select(comment => new { comment.Id, comment.Body });
 
-            Console.WriteLine($"Found {issueComments} comments for PR {prNumber}");
+            var prComments = (await connection.Run(query)).ToList();
+        
+            Console.WriteLine($"Found {prComments.Count} comments for PR {prNumber}");
 
-            var count = 0;
-            foreach (var issueComment in issueComments)
+            var updated = false;
+            foreach (var prComment in prComments)
             {
-                if (issueComment.IsMinimized || ! issueComment.Body.StartsWith(prefix))
+                if (! prComment.Body.StartsWith(title))
```

Comment:
nit: I think we could probably tweak this loop slightly. Instead of looping through _all_ comments, we could stop as soon as we have updated one.

If we go with that approach, we would also need to skip any hidden comments, so that we only update the "visible comment". 

I think a simple approach would be to include `comment.IsMinimized` in the selector above, and just skip it if it's minimized.

```suggestion
                if (prComment.IsMinimized || !prComment.Body.StartsWith(title))
```

While I'm writing this, I'm wondering if it's worth the hassle (especially if we have _two_ visible comments due to a previous bug or something) so really don't worry about this if you don't agree / can't be bothered 😆

Reply:
Great suggestion on the stopping looping.

That could happen when a bug happens, so it's technically possible. I'll add it, since it doesn't hurt.

Reply:
Awesome - once it's tested, I would say feel free to merge it, no need to wait for a second review on this one 🙂